admin/sql/updates/2021-03-02-add-user-recommendation-event-table.sql

```diff
@@ -0,0 +1,22 @@
+BEGIN;
+
+CREATE TYPE user_recommendation_event_type_enum AS ENUM('recording');
+
+CREATE TABLE recommendation.user_recommendation_event (
+  id                    SERIAL, -- PK
+  user_id               INTEGER, -- FK to "user"
+  recommendation_type   user_recommendation_event_type_enum,
+  metadata              JSONB,
```

Comment:
What do you plan to store in here?

Reply:
I haven't defined a spec yet, but for the track recommendation feature it'd store track metadata (artist name, track name, spotify ID, MBIDs) etc.